src/proto/grpc/testing/echo.proto

```diff
@@ -32,7 +32,7 @@ syntax = "proto3";
 
 import "src/proto/grpc/testing/echo_messages.proto";
 
-package grpc.testing;
```

Comment:
There is a namspace collision on grpc.testing.UnimplementedService.  If you try to do a batch build of all the protos, it fails because of this.  I've reverted this and am doing singular builds because this namspace change broke the build.